NSS-Spontaneous-Spoonbills/Sprint2
# Js editable employee

## Link to Ticket

Autolink format: hashtag and number, ex: #1094

Closes no tickets

## Description of Proposed Changes
1. When an employee has been created, admin users have permission to update the employee field.
1. Does not include removing trainings from user.

## Steps to Test

Outline the steps to test
1. Pull down this branch
1. Make migrations
1. Migrate 
1. Run server
1. Create two new departments
1. Create two new computers
1. Create two employees
1. Redirect back to employee tab
1. Check and see if you can update an employee
1. Save changes
1. Also, you should be able to load up 'localhost:/WorkforceManagement'
1. Added home button to redirect to home page.
1. Give me a :shipit: 
```
git fetch --all
git checkout js_editable_employee
```

## Impacted Areas in Application
1. WorkforceManagement/views/Employee_View.py
1. WorkforceMangement/urls.py


## Mentions @username
@binauralbeat @obscure-shadow @CashewRose @MrErin 
Tag users that need to review this code

## Testing

No testing was done on this PR.

## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
